### ISSUE TITLE
SSL issue

### ISSUE THREAD

**Issue by nhoizey** [aarongustafson/jekyll-webmention_io#72]
On my Mac (macOS High Sierra 10.13), I get this error on each attempt to get mentions from webmention.io:

```
[jekyll-webmention_io] Got an error checking https://webmention.io/api/mentions?target[]=https://nicolas-hoizey.com/2017/11/medium-is-only-an-edge-server-of-your-posse-cdn-your-own-blog-is-the-origin.html&target[]=http://nicolas-hoizey.com/2017/11/medium-is-only-an-edge-server-of-your-posse-cdn-your-own-blog-is-the-origin.html&since_id=472395&perPage=9999: Connection reset by peer - SSL_connect
```

I don't think I've changed anything in my configuration.

I tried [these instructions](https://github.com/fastlane/fastlane/issues/6758#issuecomment-256490784) from a similar issue, but it didn't help.

**Comment by aarongustafson** [comment]
Check in with @aaronpk. I think that's gotta be an SSL issue on his site.

**Comment by nhoizey** [comment]
Ok, thanks.

**Comment by aaronpk** [comment]
I don't know of any issues with the webmention.io SSL certificate. Did anything change about the computer this is running on? Maybe a new Ruby version? New OS updates? Chances are this is an error where the computer this is running on doesn't trust the certificate authority that issued the certificate. (Let'sencrypt)

**Comment by nhoizey** [comment]
@aaronpk I think I've made some macOS X updates indeed. I'll try to find if other people with recent version have issues with Ruby and SSL.

**Comment by mikka2061** [comment]
No updates here, and same:

jekyll-webmention_io] Got an error checking https://webmention.io/api/mentions?target[]=<targets...>: Connection reset by peer - SSL_connect

This is after a few months of working perfectly. Other than checking some stuff into _posts/ there is no change to Jekyll itself, neither gems nor SSL on the host has been updated, either. Linux (Debian).

**Comment by aaronpk** [comment]
I managed to replicate this issue and track it down. Is there a reason this library is [forcing TLSv1](https://github.com/aarongustafson/jekyll-webmention_io/blob/master/lib/jekyll/webmention_io.rb#L249)? 

With that line removed, or changing it to `:TLSv1_1` I'm able to make a request to webmention.io. No change was needed for the webmention.io server.

**Comment by aarongustafson** [comment]
Thanks @aaronpk! I'll get that fixed today. I think I was using example
code for the HTTPS connection. Bugfix forthcoming.

**Comment by aarongustafson** [comment]
This should be fixed in 2.8.5. @nhoizey can you verify the issue has gone away for you?

**Comment by nhoizey** [comment]
@aarongustafson it works, indeed, thanks a lot!
